source/index.md

```diff
@@ -92,6 +103,7 @@ See our [example notebooks repo](https://github.com/rapidsai/cloud-ml-examples)
 local
 cloud/index
 hpc
+mig
```

Comment:
I'm not sure this should be a top-level navigation item. I wonder if we need a new section like "Best practices" or something for topics like this to live under?

Reply:
Yeah, I wasn't sure where this would live either, as MIG doesn't accurately fit any of the existing sections. It's technically not a best practice either, since MIG is a specific NVIDIA/CUDA technology. Maybe something like "Other Uses", or something that presents the same idea but better phrased?

Reply:
Yeah that makes sense. I'm tempted to hit merge as is, for now, as I expect we will do a bit of a refactor before 22.12. Other similar sections may come up and inspire the name of a common place to put things.